pip-lock/PhylumExt.toml

```diff
@@ -0,0 +1,8 @@
+name = "pip-lock"
+description = "Utility for working with loose requirements files"
+entry_point = "main.ts"
+
+[permissions]
+run = ["pip", "pip3"]
+read = ["./"]
+env = ["PHYLUM_PIP_PATH"]
```

Comment:
If somebody sets `PHYLUM_PIP_PATH` to something like `/opt/homebrew/bin/pip3.10`, won't that be blocked because the path and the name of the executable are different from what's on the allowed list? I don't know if we can allow running files that are named by environment variables or if the user can explicitly authorize additional executables when invoking the extension.